server/test/services/completion/imports.ts

```diff
@@ -98,42 +98,6 @@ describe("Parser", () => {
             errors: [],
           }));
         });
-
-        it("should list sol files under a package folder in node_modules", () =>
```

Comment:
Is the idea here to swap the unit tests for the integration tests?

Reply:
The removed unit tests were related to hardhat import completions, so I thought that instead of mocking the necessary components, I'd just write integration tests which ensure that ensure the whole completion process works as expected. They slower so that's definitely a tradeoff, but I'm not running them all the time. Should I restore the unit tests and write the required mocking components to support it ?

Reply:
I would have an integration for the happy path and unit tests for the edge cases (the feature works across platform and here is my definition of works in detail).

If we only leverage the integration test suite it will get slower and slower over time and we miss out on the design pressure that comes at the component level from having to write components that can be tested.

Reply:
We got our answer on testing (add a new integration test suite against the LSP). So the question here is, are there any test cases in these unit tests that we think should make the migration to LSP integration tests - and hence should be ported to ui integration tests? Or do we think we have coverage of the import paths, in which case we are good?

Reply:
I'd say we are good, for this case I wrote at least 1 e2e test for each scenario removed from unit tests. After we have LSP integration tests I'll work on adding more thorough coverage since we'll have a much faster test suite.

Reply:
Brilliant. Then let me approve this.